### PR TITLE
Omit national cloud text.

### PR DESCRIPTION
**Pull request by andrueastman**
This PR fixes https://github.com/microsoftgraph/msgraph-metadata/issues/503

It omits the Cloud Deployments description when annotating types and methods in the metadata to improve the metadata quality.

### CODE REVIEW

File: ApiDoctor.Validation/DocFile.cs [hunk 1]
```diff
@@ -581,7 +581,8 @@ protected bool ParseMarkdownBlocks(IssueLogger issues)
                     else if (previousHeaderBlock.BlockType == BlockType.h1)
                     {
                         methodDescriptionsData.Add(block.Content);
-                        methodDescription = string.Join(" ", methodDescriptionsData.SkipWhile(x => x.StartsWith("Namespace:"))).ToStringClean();
+                        // make sure we omit the namespace description as well as the national cloud deployments paragraphs
+                        methodDescription = string.Join(" ", methodDescriptionsData.Where(static x => !x.StartsWith("Namespace:") && !x.Contains("[national cloud deployments](/graph/deployments)"))).ToStringClean();
```
Comment (baywet) [inline ApiDoctor.Validation/DocFile.cs:585]: string comparisons ?

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — APPROVED** [review]

File: ApiDoctor.Validation/DocFile.cs (hunk 1 above)
Comment (andrueastman) [reply ApiDoctor.Validation/DocFile.cs]: 🤦🏼. Resolved in latest update

**Comment by baywet** [comment]
I don't think I have merge privileges on this repo, we'll need @BarryShehadeh 's help. Don't hesitate to ping him on teams if he doesn't follow up promptly, he is usually responsive but can be busy at times :)

**Comment by andrueastman** [comment]
> I don't think I have merge privileges on this repo, we'll need @BarryShehadeh 's help. Don't hesitate to ping him on teams if he doesn't follow up promptly, he is usually responsive but can be busy at times :)

Sorry @baywet, saw this too late. I do have the privileges to merge. 

Happy to update anything in a separate PR in case of anything @BarryShehadeh

**Comment by baywet** [comment]
no need to bother him then, it was a small change. Didn't know you had the permission.

**Pull request merged**